packages/ui/src/api.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios'
 
 import { API_BASE_URL } from './constants'
 
-export class WittyCreaturesApi {
+export class WittyApi {
```

Comment:
What do you think about rename the file as `ApiService.ts`
```suggestion
export class ApiService {
```